src/datasource-zabbix/module.ts

```diff
@@ -19,8 +19,8 @@ ZabbixQueryController.templateUrl = 'datasource-zabbix/partials/query.editor.htm
 ZabbixDSConfigController.templateUrl = 'datasource-zabbix/partials/config.html';
 
 loadPluginCss({
-  dark: 'plugins/alexanderzobnin-zabbix-app/css/grafana-zabbix.dark.css',
-  light: 'plugins/alexanderzobnin-zabbix-app/css/grafana-zabbix.light.css'
+  dark: 'plugins/iiris-zabbix-triggers-panel/css/grafana-zabbix.dark.css',
+  light: 'plugins/iiris-zabbix-triggers-panel/css/grafana-zabbix.light.css'
```

Comment:
Path seems a bit weird.

Reply:
Path should be ok. But something else is wrong anyway, I just tried and the datasource doesn't work.

Reply:
I was just referring to "triggers-panel" - seems to me this more global css than that. 😄